### PR TITLE
Dfa doc

### PR DESCRIPTION
**Pull request by matthiasto**
- Working on the dfa appendix doc (correcting typos and detailing the numerics part)
- Adding sphinxcontrib.bibtex for managing bibliography (this means we can directly use the .bib from latex) use :cite:`Zw2000,ZwKlSa2003` to refer to some articles. The biblio now only appears in the section references at the end of the documentation. Maybe it could be nice to have a local one at the end of the page the user is reading.

### CODE REVIEW

File: avaframe/out3Plot/outAIMEC.py [hunk 1]
```diff
@@ -386,7 +386,7 @@ def resultVisu(cfgPath, cfgFlags, rasterTransfo, resAnalysis, plim):
     # Get input data
     fnames = cfgPath['pressurefileList']
 
-    flag = cfgFlags['typeFlag']
+    flag = float(cfgFlags['typeFlag'])
```
Comment (fso42) [inline avaframe/out3Plot/outAIMEC.py:389]: Why the float here? string or int should be fine, so no conversion is necessary, right?

File: requirements.txt [hunk 2]
```diff
@@ -4,3 +4,4 @@ pyshp
 scipy
 cmocean
 seaborn
+sphinxcontrib-bibtex
```
Comment (fso42) [inline requirements.txt:7]: Could this be moved to docs/requirements.txt? 
To keep this requirements as clean as possible. A user who does not build the documentation, wanting only to run the model, does not need this module...

File: avaframe/out3Plot/outAIMEC.py (hunk 1 above)
Comment (fso42) [reply avaframe/out3Plot/outAIMEC.py:389]: Ah, i forgot about the missing type conversion of the ini system

File: .github/workflows/runTestSinglePython.yml [hunk 3]
```diff
@@ -26,6 +26,7 @@ jobs:
         pip install flake8 pytest
         pip install pytest-cov
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f docs/requirements.txt ]; then pip install -r docs/requirements.txt; fi
```
Comment (fso42) [inline .github/workflows/runTestSinglePython.yml:29]: Is this really needed? As the test is not building the docs... and readthedocs knows about the docs/req...  anyways...

Comment (matthiasto) [reply .github/workflows/runTestSinglePython.yml:29]: I havent tried without but if you say reads the docs knows about it, I remove it

Comment (fso42) [reply .github/workflows/runTestSinglePython.yml:29]: If readthedocs does not automatically find it, we can point to it in the Advanced settings of the avaframe project on rtd...

**Pull request merged**